libraries/CAN/test/unit/src/framework/arduino/Test_CAN.cpp

```diff
@@ -0,0 +1,85 @@
+// std includes
+
+// test includes
+#include "Test_includes.hpp"
+
+// project includes
+
+
+// variables used in the tests below that have to be accessed in the setup and tear down methods
```

Comment:
Commented code without explanation.

Reply:
These were all for Unity - trying something at the time

Reply:
In general, what I mean is that we can have disabled (commented out) code release. but it should be explained why is it commented, and when to enable that code, etc.
Otherwise I would see as code needs to be cleaned up.

Reply:
make sense. I remove it for this release, on Thursday Jens will show me the new setup and then I will start to add the unit test properly